bazarr/subtitles/upgrade.py

```diff
@@ -303,21 +362,66 @@ def get_upgradable_movies_subtitles():
         .subquery()
 
     minimum_timestamp, query_actions = get_queries_condition_parameters()
+    logging.debug(f"Minimum timestamp used for subtitles upgrade: {minimum_timestamp}")
+    logging.debug(f"Those actions are considered for subtitles upgrade: {query_actions}")
 
     upgradable_movies_conditions = [(TableHistoryMovie.action.in_(query_actions)),
                                     (TableHistoryMovie.timestamp > minimum_timestamp),
                                     TableHistoryMovie.score.is_not(None),
                                     (TableHistoryMovie.score < 117)]
```

Comment:
Magic number "117" is 120 - 3?
Just guessing... 😉

Reply:
You're getting good at it! ;-)